src/state/middleware/title.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */
+const siteTitle = document.title;
+export default store => next => action => {
+  const result = next(action);
+  if (action.type === 'NOW_PLAYING') {
```

Comment:
Instead of using strings, can we create constants that we can use throughout the whole app?
`if (action.type === ACTIONS.NOW_PLAYING)`

Reply:
Good point, I'll take a look.

Reply:
Updated, thanks